src/react/databrowser/objects/__tests__/Metadata.test.jsx

```diff
@@ -0,0 +1,102 @@
+import * as s3objects from '../../../actions/s3object';
+import { AddButton, SubButton } from '../../../ui-elements/EditableKeyValue';
+import Metadata from '../details/Metadata';
+import { OBJECT_METADATA } from '../../../actions/__tests__/utils/testUtil';
+import React from 'react';
+import { reduxMount } from '../../../utils/test';
+
+describe('Metadata', () => {
+    const putObjectMetadataMock = jest.spyOn(s3objects, 'putObjectMetadata').mockReturnValue({ type: '' });
+    const optionLabels = ['cache-control', 'content-disposition', 'content-encoding', 'content-language', 'content-type', 'expires', 'website-redirect-location', 'x-amz-meta'];
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('Metadata should render', () => {
+        const { component } = reduxMount(<Metadata objectMetadata={OBJECT_METADATA}/>);
+
+        expect(component.find(Metadata).isEmptyRender()).toBe(false);
+    });
+
+    it('should render by default a SelectBox with empty values in each input when there are no key/value present', () => {
+        const { component } = reduxMount(<Metadata objectMetadata={OBJECT_METADATA}/>);
+
+        expect(component.find('input[name="mdKeyType"]').prop('value')).toBe('');
```

Comment:
We can first check that only one "row" is returned before checking its value.

Reply:
I already check it if you look two lines below:
expect(component.find('SelectBox')).toHaveLength(1);
but maybe it was not clearer so now I check if only one item is rendered before checking values and I'm looking for Item component and not SelectBox Component